dev/release-aux/openssl-announce-pre-release.tmpl

```diff
@@ -0,0 +1,49 @@
+
+   OpenSSL version $release ($LABEL)
+   ===========================================
+
+   OpenSSL - The Open Source toolkit for SSL/TLS
+   https://www.openssl.org/
+
+   OpenSSL $MAJOR.$MINOR.$PATCH is currently in $LABEL.
```

Comment:
Similarly here. Not sure about showing the patch number. This ends up as:

>  OpenSSL 3.0.0 is currently in alpha.

To my eye, this is better as:

> OpenSSL 3.0 is currently in alpha

Reply:
Here it makes sense because this is rather the name of the release stream.

Reply:
Related to that, @t8m, would you rather the release branch be called `openssl-3.0.x` or would you prefer `openssl-3.0`?  I do prefer the former, but can go with the latter if that's what everyone else prefers.

Reply:
I do not really have a preference here, both are OK.

Reply:
I think I'll keep `.x`, as it makes it more distinct